tasks/src/reinforcement-learning/about.md

```diff
@@ -65,19 +70,32 @@ For solving this problem DRRN(deep reinforcement relevance network) was proposed
 
 The deep Reinforecement Learning finds application in a chatbot dialogue. Conversations are simulated using two virtual agents and the quality is improved in progressive iterations.
 
-Robots in industries or healthcare working towards reducing manual intervention use reinforcement learning to map natural language instructions to sequences of executable actions. During training, the learner repeatedly constructs action sequences, executes those actions, and observes the resulting rewards. A reward function works in the backend that defines the quality of these executed actions.
 
-### Robotics 
+## Glossary

```

Comment:
I think it would be good to put this section above Use Cases given it's essential to understand use cases.

Reply:
Can you address this comment?